[BE] Endpoint for activity type list

MihaiDruta opened the pull request:
[BE] Endpoint for activity type list
https://trello.com/c/F53JUxt5/75-be-endpoint-for-activity-type-list

Added endpoint for list of activity types sorted alphabetically.

Commit:
[BE] Endpoint for activity type list
https://trello.com/c/F53JUxt5/75-be-endpoint-for-activity-type-list

Added endpoint for list of activity types sorted alphabetically.

SpartakusMd commented on src/Controller/ActivityTypeController.php:
```diff
@@ -0,0 +1,78 @@
+
+/**
+ * ActivityType controller.
+ * @Route("/api/types", name="activityTypes")
```
`/api/activity-types`

SpartakusMd commented on src/Controller/ActivityTypeController.php:
```diff
@@ -0,0 +1,78 @@
+     */
+    private $serializer;
+
+    public function __construct(TypeRepository $typeRepository, SerializerInterface $serializer)
```
Shouldn't `TypeRepository` be `ActivityTypeRepository`? I thought it was renamed.

Commit:
[BE] Endpoint for activity type list
https://trello.com/c/F53JUxt5/75-be-endpoint-for-activity-type-list

Renamed TypeRepository to ActivityTypeRepository. Renamed route to api/activity-types
